packages/atomic-angular/project.json

```diff
@@ -7,7 +7,7 @@
   "targets": {
     "cached:build": {
       "outputs": ["{projectRoot}/projects/atomic-angular/dist"],
-      "dependsOn": ["^build", "clean"],
+      "dependsOn": ["^build"],
```

Comment:
The clean task doesn't exist (and never did AFAIK), so it was just getting skipped.